README.md

```diff
@@ -128,6 +98,8 @@ Join a new worker node to the cluster by running:
 k0s worker "superlongtokenfrompreviousphase"
 ```
 
+The client config (for kubectl or lens) can be found in `/var/lib/k0s/pki/admin.conf` on the controller.
+
```

Comment:
We should also point out that "localhost" needs to be replaced if hosted elsewhere than cloud.